<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SecureChat by Li_Big_A</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(219, 230, 255)</string>
  </property>
  <widget class="QLineEdit" name="chatInput">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>301</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sendButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>390</y>
     <width>91</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(213, 255, 149)</string>
   </property>
   <property name="text">
    <string>전송</string>
   </property>
  </widget>
  <widget class="QLabel" name="chatLog">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>401</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 255, 127)</string>
   </property>
   <property name="text">
    <string>chatLog</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>325</x>
     <y>10</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>메뉴</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>대화 검색</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>대화 캡쳐</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>파일 전송</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>채팅방 설정</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>채팅방 나가기</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>닫기</string>
    </property>
   </item>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>490</y>
     <width>401</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QToolButton" name="toolButton_4">
      <property name="accessibleName">
       <string>\</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="toolButton_3">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="toolButton_2">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="toolButton">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
